src/twisted/internet/test/test_inlinecb.py

```diff
@@ -202,13 +202,22 @@ def _noReturn():
     def testReturnValue(self):
         """Ensure that returnValue works."""
```

Comment:
Maybe rename the tests as one of the main goal of this test is to check for deprecation.

```suggestion
    def testReturnValueDeprecated(self):
        """C{returnValue} is now deprecated but continues to be available."""
```

Reply:
Makes sense, thanks.